ddev/src/ddev/cli/release/branch/create.py

```diff
@@ -0,0 +1,53 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from __future__ import annotations
+
+import re
+from typing import TYPE_CHECKING
+
+import click
+
+if TYPE_CHECKING:
+    from ddev.cli.application import Application
+
+BRANCH_NAME_PATTERN = re.compile(r'^\d+\.\d+\.x$')
+GITHUB_LABEL_COLOR = '5319e7'
+
+
+@click.command
+@click.pass_obj
+@click.argument('branch_name')
+def create(app: Application, branch_name):
```

Comment:
Could we rename this to `agent_version`? That's what our validation is about.

Reply:
That's exactly what's the validation is about, we check if it's something like `7.52.x`